LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) Bo Peng and the University of Texas MD Anderson Cancer Center
```

Comment:
the license ownership seems strange no?

Reply:
Sorry, I simply copied my LICENSE file without looking. Will revise.

Reply:
Fixed. Good try to steal other's work though. :-)

Reply:
But somebody needs to claim copyright ;)

Reply:
I think "calysto developers" is the right name

Reply:
https://github.com/Calysto/metakernel/blob/master/LICENSE.txt uses 

```
Copyright (c) 2014-2019, IPython Kernel Development Team and Metakernel Development Team
```

Reply:
That one did use some code from IPython.  I think Calysto is more appropriate here.

Reply:
@blink1073 I have added a license line but it can be easier for you to add the file with proper term by yourself...

Reply:
I am afk for the night, "Calysto Development Team" is the term I would use